diff --git a/pkg/freetype/face.zig b/pkg/freetype/face.zig
index eea3c685..f57f9af0 100644
--- a/pkg/freetype/face.zig
+++ b/pkg/freetype/face.zig
@@ -33,8 +33,8 @@ pub const Face = struct {
 
     /// A macro that returns true whenever a face object contains an ‘sbix’
     /// OpenType table and outline glyphs.
-    pub fn hasSBIX(self: Face) bool {
-        return c.FT_HAS_SBIX(self.handle);
+    pub fn hasSBIX(_: Face) bool {
+        return false;
     }
 
     /// A macro that returns true whenever a face object contains some
diff --git a/pkg/oniguruma/errors.zig b/pkg/oniguruma/errors.zig
index d63a481b..69cd2d51 100644
--- a/pkg/oniguruma/errors.zig
+++ b/pkg/oniguruma/errors.zig
@@ -44,7 +44,6 @@ pub const Error = error{
     RetryLimitInMatchOver,
     RetryLimitInSearchOver,
     SubexpCallLimitInSearchOver,
-    DefaultEncodingIsNotSet,
     SpecifiedEncodingCantConvertToWideChar,
     FailToInitialize,
     InvalidArgument,
@@ -67,7 +66,6 @@ pub const Error = error{
     EndPatternWithUnmatchedParenthesis,
     EndPatternInGroup,
     UndefinedGroupOption,
-    InvalidGroupOption,
     InvalidPosixBracketType,
     InvalidLookBehindPattern,
     InvalidRepeatRangePattern,
@@ -130,7 +128,6 @@ const error_code_map: []const struct { Error, c_int } = &.{
     .{ Error.RetryLimitInMatchOver, c.ONIGERR_RETRY_LIMIT_IN_MATCH_OVER },
     .{ Error.RetryLimitInSearchOver, c.ONIGERR_RETRY_LIMIT_IN_SEARCH_OVER },
     .{ Error.SubexpCallLimitInSearchOver, c.ONIGERR_SUBEXP_CALL_LIMIT_IN_SEARCH_OVER },
-    .{ Error.DefaultEncodingIsNotSet, c.ONIGERR_DEFAULT_ENCODING_IS_NOT_SET },
     .{ Error.SpecifiedEncodingCantConvertToWideChar, c.ONIGERR_SPECIFIED_ENCODING_CANT_CONVERT_TO_WIDE_CHAR },
     .{ Error.FailToInitialize, c.ONIGERR_FAIL_TO_INITIALIZE },
     .{ Error.InvalidArgument, c.ONIGERR_INVALID_ARGUMENT },
@@ -153,7 +150,6 @@ const error_code_map: []const struct { Error, c_int } = &.{
     .{ Error.EndPatternWithUnmatchedParenthesis, c.ONIGERR_END_PATTERN_WITH_UNMATCHED_PARENTHESIS },
     .{ Error.EndPatternInGroup, c.ONIGERR_END_PATTERN_IN_GROUP },
     .{ Error.UndefinedGroupOption, c.ONIGERR_UNDEFINED_GROUP_OPTION },
-    .{ Error.InvalidGroupOption, c.ONIGERR_INVALID_GROUP_OPTION },
     .{ Error.InvalidPosixBracketType, c.ONIGERR_INVALID_POSIX_BRACKET_TYPE },
     .{ Error.InvalidLookBehindPattern, c.ONIGERR_INVALID_LOOK_BEHIND_PATTERN },
     .{ Error.InvalidRepeatRangePattern, c.ONIGERR_INVALID_REPEAT_RANGE_PATTERN },
