From: Mike Kasberg <kasberg.mike@gmail.com>
Description: Patch libbzip2 to libbz2 for debian
diff -Nru ghostty-1.0.1.orig/build.zig ghostty-1.0.1/build.zig
--- ghostty-1.0.1.orig/build.zig
+++ ghostty-1.0.1/build.zig
@@ -1064,7 +1064,7 @@
         step.root_module.addImport("freetype", freetype_dep.module("freetype"));
 
         if (b.systemIntegrationOption("freetype", .{})) {
-            step.linkSystemLibrary2("bzip2", dynamic_link_opts);
+            step.linkSystemLibrary2("bz2", dynamic_link_opts);
             step.linkSystemLibrary2("freetype2", dynamic_link_opts);
         } else {
             step.linkLibrary(freetype_dep.artifact("freetype"));
